app/components/provider_interface/user_org_permission_breakdown_component.html.erb

```diff
@@ -0,0 +1,21 @@
+<% if permitted_by_providers.any? %>
+  <p class="govuk-body">
+    It currently applies to courses you work on with all of your partner organisations:
```

Comment:
The content for this should change based on whether all partners fall in to one category, or if they fall into a mixture of the categories. See the logic & content in the prototype code: https://github.com/DFE-Digital/manage-teacher-training-applications-prototype/blob/8bb93227ebd80d8aeb80e5643e26461bfade765b/app/views/_components/applicable-permissions-inline/template.njk